app/src/main/java/com/ucmobiledevelopment/freeways/ReportIncidentActivity.kt

```diff
@@ -18,7 +18,7 @@ class ReportIncidentActivity : ComponentActivity() {
     override fun onCreate(savedInstanceState: Bundle?) {
         super.onCreate(savedInstanceState)
         setContent {
-            FreeWaysTheme() {
+            FreeWaysTheme{
```

Comment:
A good suggestion, though it appeared in a previous review. When that happens, first in wins.